src/sidebar/store/modules/frames.js

```diff
@@ -18,6 +18,7 @@ import { createStoreModule } from '../create-store';
  *   - Sub-frames will all have a id (frame identifier) set. The main frame's id is always `null`
  * @prop {DocumentMetadata} metadata - Metadata about the document currently loaded in this frame
  * @prop {string} uri - Current primary URI of the document being displayed
+ * @prop {boolean} [isAnnotationFetchComplete]
```

Comment:
The data model could be simplified here by making this property mandatory, in which case it will need to be set to `false` as part of the `connectFrame` action.